2024/utils/constants.js

```diff
@@ -21,7 +21,7 @@ export const SPONSORS_2024 = {
     {
       name: "Pulitzer Center",
       href: "https://pulitzercenter.org/",
-      logo: "/2024/sponsors/pulitzer-center-logo.jpg",
+      logo: "/2024/sponsors/pulitzer-center-logo.webp",
```

Comment:
Thanks for the fixes @GeoXhacker , on the Pulitzer logo, we'd kept it with a white background just as it was in their brand guide due to guidelines that are usually set by the organisations around modification of brand items...nevertheless will revert back on this particular one.

Reply:
Oh I completely understand. It alright matey.

Reply:
made some followups, and seems okay to have the logo without the background 👍